src/Platform/Microsoft.Testing.Platform/Hosts/CommonTestHost.cs

```diff
@@ -53,6 +75,15 @@ public async Task<int> RunAsync()
         }
         finally
         {
+#if NETCOREAPP
+            if (dotnetTestPipeClient is not null)
+            {
+                await dotnetTestPipeClient.DisposeAsync();
+            }
+#else
+            dotnetTestPipeClient?.Dispose();
+#endif
```

Comment:
You can use the helper for this case, it simplifies reading

```suggestion
            await DisposeHelper.DisposeAsync(dotnetTestPipeClient);
```